cmd/meroxa/root/apps/apps.go

```diff
@@ -24,7 +24,7 @@ import (
 
 type Apps struct{}
 
-const LanguageNotSupportedError = "Currently, we support \"javascript\", \"golang\", and \"python\""
+const LanguageNotSupportedError = "Currently, we support \"javascript\", \"golang\", \"python\", and \"ruby (beta)\" "
```

Comment:
Not part of this pull-request, but I think we should start centralizing language support in turbine-core so we remove these from the CLI. I have a TODO item for this, so I'll make sure it gets done.

Reply:
Sounds good! Thanks Raul!